src/dmd/dversion.d

```diff
@@ -199,3 +199,16 @@ extern (C++) final class VersionSymbol : Dsymbol
         v.visit(this);
     }
 }
+
+private Module moduleOf(ScopeDsymbol sds)
```

Comment:
A function that nobody calls and has no documentation?

Reply:
The joy of buggering around with git. I'll fix this up soon.

Reply:
There we go.